easybuild/easyblocks/g/gcc.py

```diff
@@ -79,8 +79,8 @@ def __init__(self, *args, **kwargs):
             raise EasyBuildError("Using ISL bundled with CLooG is unsupported in >=GCC-4.8.0. "
                                  "Use a seperate ISL: set withisl=True")
 
-        # I think ISL without CLooG has no purpose in GCC...
-        if self.cfg['withisl'] and not self.cfg['withcloog']:
+        # I think ISL without CLooG has no purpose in GCC < 5.0.0 ...
+        if self.version < LooseVersion("5.0.0") and self.cfg['withisl'] and not self.cfg['withcloog']:
```

Comment:
hmm, hold on...

why isn't this:

```python
if LooseVersion(self.version) < LooseVersion('5.0') ...
```

We should be using `LooseVersion` *everywhere* where we check versions.

It may work now, but that's just by accident, I fear...